cmd/dep/godep_importer_test.go

```diff
@@ -18,6 +18,189 @@ import (
 
 const testGodepProjectRoot = "github.com/golang/notexist"
 
+func TestGodepConfig_Convert(t *testing.T) {
+	testCases := map[string]struct {
+		json                   godepJSON
+		expectConvertErr       bool
+		matchPairedVersion     bool
+		projectRoot            gps.ProjectRoot
+		notExpectedProjectRoot *gps.ProjectRoot
```

Comment:
I'm having it as a pointer to represent that it's not set. So that we can distinguish between:

- A project root `"github.com/a/b/c"`
- An empty project root `""`
- A test case that does not have a `notExpectedProjectRoot` test case

In other words: If it's `nil` it won't be part of the test case, while making it possible to have _any_ other value when we use it for testing.